connection_manager.go

```diff
@@ -407,21 +426,24 @@ func (n *connectionManager) shouldSwapPrimary(current, primary *HostInfo) bool {
 	// If we are here then we have multiple tunnels for a host pair and neither side believes the same tunnel is primary.
 	// Let's sort this out.
 
-	if current.vpnIp.Compare(n.intf.myVpnNet.Addr()) < 0 {
+	//TODO: current.vpnIp should become an array of vpnIps
+	if current.vpnAddrs[0].Compare(n.intf.myVpnAddrs[0]) < 0 {
```

Comment:
I think we will want to find the matching network here instead of using the first vpn addr like in https://github.com/slackhq/nebula/pull/1261

Reply:
Actually `hostinfo.vpnAddrs` should only contain matching addrs, so this code should be fine

Note, we need to make `hostinfo.vpnAddrs` only contain matching addres